FrontEnd/src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+import componentCSS from './Button.scss';
+
+class Button extends Component {
+
+
+    render() { 
+
+        console.log(this.props);
+
+    return ( <button onClick={() => this.props.onclick()} className={this.buttonClass()}>{this.props.Text}</button> );
```

Comment:
with this, did you mean to declare a custom button, dropping the framework? O.o
http://react-materialize.github.io/react-materialize/?path=/story/components-button--default